pootle/static/css/editor.css

```diff
@@ -1728,3 +1734,16 @@ td.translate-delete-terminology input
 {
     float: right;
 }
+
+
+.js-auto-matched-translation textarea
+{
+    color: #aaa;
+}
+
+.js-auto-match-msg
```

Comment:
I realize that it has been done like this a few times. But the agreement years ago was to have separate classes for JS and CSS code, even if that meant having two classes in the same element, like

```html
<p class="something js-something">Hi</p>
```